x-pack/plugins/alerting/server/saved_objects/migrations/8.6/index.ts

```diff
@@ -36,7 +36,8 @@ const getLastRun = (attributes: RawRule) => {
 
   return {
     outcome,
-    outcomeMsg: warning?.message || error?.message || null,
+    outcomeMsg:
+      (error?.message && [error?.message]) || (warning?.message && [warning?.message]) || null,
```

Comment:
my only worry is that, I think I will prefer to not change this migrations and leave the way it is and maybe create a new migration or just make sure that the API return  array on the outcomeMsg field. Touching existing migration is not the way to go, it is kind of too late